stairctl/src/main/resources/application.yml

```diff
@@ -7,3 +7,15 @@ stairctl:
   host: ${STAIRCTL_HOST:127.0.0.1}
   port: ${STAIRCTL_PORT:5432}
   dbname: ${STAIRCTL_DBNAME:stairwaylib}
+
+spring:
+  shell:
+    interactive:
+      enabled: true
+    script:
+      enabled: true
+
+# Disable logging
```

Comment:
While looking the logging dependencies, I thought that, as a command line program, stairctl shouldn't log at all, so I added a config to `properties` to disable logging.